pom.xml

```diff
@@ -1,20 +1,42 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<groupId>nf.fr.k49</groupId>
-	<artifactId>shielddb-parent</artifactId>
-	<version>0.1.1</version>
-	<packaging>pom</packaging>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>nf.fr.k49</groupId>
+    <artifactId>shielddb-parent</artifactId>
+    <version>0.1.1</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <junit.version>4.12</junit.version>
+        <mockito.version>3.1.0</mockito.version>
```

Comment:
why mockito 3.1.0? why not 3.5.13?

Reply:
You're right! I've used a configuration that I have as a reference and because I've put my focus on the implementation of the tests instead of caring about putting the latest version.... actually I think that I can remove it for now once there is no test using such library.